chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/surveillance/PendingSurveillanceRequirementEntity.java

```diff
@@ -17,10 +17,13 @@
 import javax.persistence.Table;
 
 import gov.healthit.chpl.entity.CertificationCriterionEntity;
-import gov.healthit.chpl.util.Util;
+import lombok.Getter;
+import lombok.Setter;
 
 @Entity
 @Table(name = "pending_surveillance_requirement")
+@Getter
+@Setter
```

Comment:
Yes, I originally used the Data annotation but the generated equals/hashcode (forget which) was giving some NPEs. Will add ToString since you mentioned it